packages/expandable/src/ux-expandable.ts

```diff
@@ -61,22 +47,17 @@ export class UxExpandable implements UxComponent {
 
   updateContainerHeight() {
     if (this.openBoolean) {
-      // after transition set body height to auto so that expandable children are visible
-      this.contentContainer.addEventListener("transitionend", this);
-      this.contentContainer.style.height = this.content.clientHeight + "px";
+      this.contentContainer.style.height = this.content.clientHeight + 'px';
     } else {
-      // the following line is needed because height has been restored to auto"
-      this.contentContainer.style.height = this.content.clientHeight + "px";
-      this.taskQueue.queueTask(() => {
-        this.contentContainer.style.overflow = "hidden";
-        this.contentContainer.style.height = "0";
-      });
+      this.contentContainer.style.height = '0';
```

Comment:
Doesn't it need to keep the hidden overflow to make sure its content is hidden.

Reply:
It's hidden in the css

Reply:
nice, thanks. I just realized we may need to observe the size of the content and adjust the size of this container accordingly, otherwise when the content grows, it will be clipped. or we can wait until the transition finishes and remove the fixed height

Reply:
Oh, that might be the reason I had it so complex in the first place... To support growing content

Reply:
I've reverted this change